pom.xml

```diff
@@ -83,35 +109,40 @@
       <version>1.10</version>
     </dependency>
     <dependency>
-      <groupId>org.xerial</groupId>
-      <artifactId>sqlite-jdbc</artifactId>
-      <version>3.7.2</version>
-    </dependency>
-    <dependency>
-        <groupId>commons-configuration</groupId>
-        <artifactId>commons-configuration</artifactId>
-        <version>1.6</version>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+      <version>1.10</version>
     </dependency>
     <dependency>
         <groupId>org.twitter4j</groupId>
-        <artifactId>twitter4j-core</artifactId>
-        <version>3.0.3</version>
+        <artifactId>twitter4j-stream</artifactId>
+        <version>[4.0,)</version>
```

Comment:
I think it does not hurt to use 4.0.4 or 4.0.5. They are just minor updates. However, to be sure it works, we can stay on a fixed version.

Regarding the pulling of version numbers out to properties, we can discuss this on IRC so you can clarify some doubts I have.